src/components/WorkshopPage/WorkshopPage.js

```diff
@@ -225,9 +226,15 @@ function WorkshopPage() {
 
 	return (
 		<Container maxWidth='md' style={{ marginBottom: theme.spacing(8) }}>
-			<Typography variant='h4' component='h1' className={classes.title}>
-				Workshops
-			</Typography>
+			<Box style={{ marginBottom: theme.spacing(4) }}>
+				<Typography variant='h4' component='h1' className={classes.title}>
+					Workshops
+				</Typography>
+
+				<Typography variant='body1' component='body1'>
```

Comment:
the `component` prop refers to the actually html tag or react component that is being used to render the text. In this case, I don't think that there is a body1 tag, so you should be using `body` instead.